server/lib/src/schema.rs

```diff
@@ -214,7 +214,7 @@ impl SchemaAttribute {
             // Comparing on the label.
             SyntaxType::TotpSecret => matches!(v, PartialValue::Utf8(_)),
             SyntaxType::AuditLogString => matches!(v, PartialValue::Utf8(_)),
-            SyntaxType::EcKey => matches!(v, PartialValue::Iutf8(_)),
+            SyntaxType::EcKey => matches!(v, PartialValue::PrivateBinary),
```

Comment:
I think this is a good idea, it allows us to check presence this way.

Reply:
Yeah but as I wrote it I got entangled like A LOT in ensuring I used the right `PartialValue`. Since it has such a different name from `EcKey` or something it's hard to spot where you're using the wrong PartialValue. Maybe we should introduce another variant with no data associated to it just for clarity?